### DISCUSSION TITLE
Simulation using Equivalent circuit model

### DISCUSSION THREAD

**Discussion by balaji3112** [pybamm-team/PyBaMM#2775] [Q&A] [answered]
How can I use an equivalent circuit model instead of SPM, SPMe, or DFN models for simulation?

**Comment by brosaplanella** [comment, marked as answer]
At the moment we only have a [Thevenin model](https://pybamm.readthedocs.io/en/latest/source/api/models/equivalent_circuit/thevenin.html). Here's an [example on how to use it](https://github.com/pybamm-team/PyBaMM/blob/develop/examples/scripts/run_ecm.py).
